benchmarker/cmd/ann_benchmark.go

```diff
@@ -1187,7 +1184,9 @@ func initAnnBenchmark() {
 	annBenchmarkCommand.PersistentFlags().StringVar(&globalConfig.FilterStrategy,
 		"filterStrategy", "sweeping", "Use a different filter strategy (options are sweeping or acorn)")
 	annBenchmarkCommand.PersistentFlags().IntVar(&globalConfig.ReplicationFactor,
-		"replicationFactor", 1, "Replication factor")
+		"replicationFactor", 1, "Replication factor (default 1)")
+	annBenchmarkCommand.PersistentFlags().BoolVar(&globalConfig.AsyncEnabled,
+		"asyncEnabled", false, "Enable asynchronous replication (default false)")
```

Comment:
Can you please name it `asyncReplicationEnabled` or something to not confuse with async indexing?